src/main/webapp/WEB-INF/pages/cart.jsp

```diff
@@ -0,0 +1,89 @@
+<%@ page contentType="text/html;charset=UTF-8" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ taglib prefix="tags" tagdir="/WEB-INF/tags" %>
+
+<jsp:useBean id="cart" type="com.es.phoneshop.model.cart.Cart" scope="request"/>
+
+<tags:master pageTitle="Cart">
+    <p>
+         Cart: ${cart}
+    </p>
+    <c:if test="${not empty param.message && empty errors}">
+           <div class="success">
+               ${param.message}
```

Comment:
let's display success messages in green color
when everything is black - it's not clear enough